docs/JSON-RPC.md

```diff
@@ -234,12 +234,20 @@ Results:
 
 | Name | Type | Description |
 | --- | --- | --- |
+| result.connections | number | The number of active connections. |
 | result.clients | array | The list of connected clients. |
 | result.clients[*].id | number | The client’s channel id. |
 | result.clients[*].address | string | The client’s address (ip:port). |
 | result.clients[*].name | string | The client’s name. |
 | result.clients[*].jitterBufferSize | number | The client’s jitter buffer size. |
 | result.clients[*].channels | number | The number of audio channels of the client. |
+| result.clients[*].instrumentCode | number | The id of the instrument for this channel. |
+| result.clients[*].instrumentName | number | The text name of the instrument for this channel. |
+| result.clients[*].city | string | The city name provided by the user for this channel. |
```

Comment:
Could someone please check if this returns the correct value?

I set the city to "...." but JSON-RPC just gives me:

```
{"id":1,"jsonrpc":"2.0","result":{"clients":[{"address":"127.0.0.1:22217","channels":2,"city":"","countryCode":0,"countryName":"Default","id":0,"instrumentCode":0,"instrumentName":"Nichts","jitterBufferSize":2,"name":"eto22","skillLevelCode":0,"skillLevelName":"Keins"}],"connections":1}}
```

Reply:
@Rob-NY could you please test this on your machine?

Reply:
Also I believe that instrumentName is not a number but rather a String. As mentioned below, I think - as this is translated - we should think of removing it.